include/crocoddyl/core/activations/quadratic-flat.hpp

```diff
@@ -0,0 +1,75 @@
+///////////////////////////////////////////////////////////////////////////////
+// BSD 3-Clause License
+//
+// Copyright (C) 2018-2019, LAAS-CNRS
+// Copyright note valid unless otherwise stated in individual files.
+// All rights reserved.
+///////////////////////////////////////////////////////////////////////////////
+
+#ifndef CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_FLAT_HPP_
+#define CROCODDYL_CORE_ACTIVATIONS_QUADRATIC_FLAT_HPP_
+
+#include <stdexcept>
+#include "crocoddyl/core/fwd.hpp"
+#include "crocoddyl/core/activation-base.hpp"
+#include "crocoddyl/core/utils/exception.hpp"
+#include <iostream>
```

Comment:
This is just a detailed thing.
It would nice if you could move all the system-wide header first, i.e.
```cpp
#include <stdexcept>
#include <iostream>
#include "crocoddyl/core/fwd.hpp"
#include "crocoddyl/core/activation-base.hpp"
#include "crocoddyl/core/utils/exception.hpp"
```

Note that this is a common good practice that we are following.